From: Dafydd Harries <daf@rhydd.org>
Date: Sat, 3 Dec 2016 16:58:53 -0500
Subject: [PATCH] remove build path from generated FLAC.tag file

Use sed to update paths to point to locations in /usr/include rather than
locations in the source directory at build time.
---
 doc/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/doc/Makefile.am b/doc/Makefile.am
index 593d691..b7bfd7a 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -25,6 +25,7 @@ FLAC.tag: Doxyfile
 	rm -rf api
 	mv doxytmp/html api
 	rm -rf doxytmp
+	sed -ie 's,>.*/include/,>/usr/include/,' FLAC.tag
 
 doc_DATA = \
 	FLAC.tag
